CHANGELOG.md

```diff
@@ -1,15 +1,111 @@
 ## HEAD
 
+## 0.15.0-alpha.1
+###### _Feb 23, 2016_
+
+This release includes huge improvements to the implementation
+of components and utility modules. The most important improvement
+is the removal of mixins from the library, thanks to the
+[great efforts](https://github.com/callemall/material-ui/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Anewoga+style-propable)
+of @newoga :+1:
+
+There are also improvements to the unit testing infrastructure. We own this
+great improvement to @nathanmarks, thanks a lot :+1:.
+
 ##### Breaking Changes
-- Remove the deprecated API of `0.14.x`. (#3108)
-- [FloatingActionButton] Fix:className now set to root element(#2310)
-- [RaisedButton] Fix:className now set to root element(#3122)
-- [LeftNav] Fix:className and `style` now set to root element(#3322)
+- [Cleanup] Remove the deprecated API of `0.14.x`. (#3108)
+- [Core] Remove the `window-listenable` mixin (#3334)
+- [Core] Remove `context-pure` mixin (#3331)
+- [Core] Remove `click-awayable` mixin (#3360)
+- [Core] Utilize keycode library and remove `key-code` util (#3371)
+- [FloatingActionButton] `className` is now set on the root element (#2310)
+- [RaisedButton] `className` is now set on the root element (#3122)
+- [LeftNav] `className` and `style` are now set on the root element (#3322)
 - [Colors] Removed default export in favor of singular exports (#2825) <br>
 **Note** This can be temoporary worked around by changing <br>
```

Comment:
Actually, if two changes are byte-wise equal git won't rule it out as a conflict. so, even if I do change it, the conflict won't occur.